### PR TITLE
README updates

### PR DESCRIPTION
**Pull request by coffeepac**
addresses why k2 and to use k2cli first

Explicit issues addressed:
why k2:  https://github.com/samsung-cnct/k2/issues/46
k2 or kraken?  https://github.com/samsung-cnct/k2/issues/64
explicit user quick start:  https://github.com/samsung-cnct/k2/issues/95 (kind of cheating but it points to k2cli)

### CODE REVIEW

**Comment by mikeln** [comment]
lgtm

File: README.md [hunk 1]
```diff
@@ -42,14 +59,14 @@ You will need the following installed on your machine:
...
-The easiest way to get started with K2 is to use a K2 container image
+The easiest way to get started with K2 directly is to use a K2 container image
 
 `docker pull quay.io/samsung_cnct/k2:latest`
 
 ## Preparing the environment  
   
 ### Initial K2 Directory
-If this is your first time using kraken, use k2 docker image to generate a 'sensible defaults' configuration:
+If this is your first time using kraken, use k2 docker image to generate a 'sensible defaults' configuration (this assumes AWS is the infrastructure provider):
```
Comment (leahnp) [inline README.md]: Should line #69 say, "using K2" instead of "kraken"?

Comment (coffeepac) [inline README.md]: yes.  ein moment.

**Comment by leahnp** [comment]
I think a "Who Should Use K2" or "What K2 Can Provide Your Project" section outlining target demographic and problems K2 specifically solves for users could be really useful. I see section this targeting a developer or project manager researching Kubernetes cluster level management options and allows us to clearly define why K2 is a good option.

**Comment by coffeepac** [comment]
@leahnp I will add another 'graph to the 'What is K2' block.  possibly as a sub header.  thanks.

@spiffxp chasing down case mixups.  
For 'what sort of cluster does the default config represent' question, what is insufficient about the second sentence in the first 'graph of the What is K2 section?

File: README.md [hunk 2]
```diff
@@ -3,7 +3,24 @@ Deploy a __Kubernetes__ cluster on top of __CoreOS__ using __Terraform__  and __
 
 [![Docker Repository on Quay](https://quay.io/repository/samsung_cnct/k2/status "Docker Repository on Quay")](https://quay.io/repository/samsung_cnct/k2)
 
+## What is K2
+K2 is an orchestration and cluster level management system for [Kubernetes](https://kubernetes.io).  Its goal is to, by default, create production scale Kubernetes
+clusters on a range of platforms.  It boasts a large configuration file for extensive tweaking for your environment and use case.  Especially if you are getting 
+started and don't need a HA production level cluster right away.
```
Comment (spiffxp) [inline README.md]: @coffeepac this is the phrase that confused me about what the default cluster represents.  Could we change it to say what the default is, and then the variety of places/ways you could tweak it?

Comment (spiffxp) [inline README.md]: To me, this i_implies_ that the default is not an HA production level cluster, but you could tweak it to be as such.  But what is the default to start with?

Comment (spiffxp) [inline README.md]: I reading the sentence above this that use the words production and default in the same sentence.  Its goal is.  But does it.  Maybe just needs some firming.

Comment (coffeepac) [inline README.md]: Firmed it up.

**Comment by coffeepac** [comment]
@leahnp added a 'What is K2 for' section.  @spiffxp do you mind giving that a read as well?  I think you'll find it interesting.

**Comment by spiffxp** [comment]
I was going to make some mocking comment about how there were 9 graphs of text before the actual code, but then I realized that's what this PR is about.  Adding those 9 graphs.  Because the code after it doesn't explain our intent.  And I can't fault that, so LGTM.

**Comment by leahnp** [comment]
LGTM

**Comment by maratoid** [comment]
LGTM

**Pull request merged**